### PR TITLE
Portal 1324 & 1752:  sort submissions and pagination

### PR DESCRIPTION
**Pull request by tbolt**
Related to the following tickets: 

https://cdc-dexops.atlassian.net/browse/PORTAL-1752
https://cdc-dexops.atlassian.net/browse/PORTAL-1324

Adding sorting and pagination `getFileSubmissions`

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -7,3 +7,16 @@
 .cdc-react .dropdown-items {
   overflow: scroll;
 }
+
+.cdc-react .sort-arrows {
+  margin-left: 5px; /* Hack until we replace the sort icons */
+  margin-right: 5px; /* Hack until we replace the sort icons */
```
Comment (sublettk) [inline src/App.css:13]: Can you add a TODO or FIX to the comment so it gets picked up by the IDE to address later?

File: src/components/Table.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      case "status":
+        return "sm";
+      case "timestamp":
+        return "md";
+      case "details":
+        return "sm";
+      default:
+        return "md";
+    }
+  };
```
Comment (sublettk) [inline src/components/Table.tsx:33]: Refactor this to not use a switch statement. `sm` is being returned twice which is a sign that a switch statement may not be the best use case for this problem.

**Review by sublettk — CHANGES_REQUESTED** [review]

**Review by ZacharyRizer — APPROVED** [review]
LGTM

File: src/components/Table.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+function PortalTable<TData>({ table }: PortalTableProps<TData>) {
+  const getColSize = (field: string) => {
+    if (field === "status" || field === "details") {
+      return "sm";
+    }
+    if (field === "timestamp") {
+      return "md";
+    }
+    return "md";
+  };
```
Comment (ZacharyRizer) [inline src/components/Table.tsx:28]: Don't need the second if statement here.

**Review by sublettk — APPROVED** [review]
LGTM!

**Pull request merged**